lib/googleDrive/index.js

```diff
@@ -0,0 +1,185 @@
+import google from 'googleapis';
+import multiparty from 'multiparty';
+import googleAuth from './googleAuth';
+import {
+  FieldValidationError,
+} from '../tools';
+
+let drive;
+
+export function setup(googleApi = {
```

Comment:
not sure defaults are needed here, what the purpose? Also pls add jsdoc, it should contain a note about the function must be invoked before others etc